step.sh

```diff
@@ -168,6 +199,46 @@ function device_farm_run {
     local run_response=$(aws devicefarm schedule-run "${run_params[@]}")
```

Comment:
If you are depending on `run_response` to be in json format please make it explicit by passing `--output=json` to `aws devicefarm schedule-run`